src/util/log.ts

```diff
@@ -36,26 +79,30 @@ export default class Logger {
   }
   
   debug(...args:  any[]) {    
-    if (logLevelSettings[this._source] >= LogLevel.Debug) {
-      console.debug(args)
+    let allowedLevel = currentLogLevelSettings[this._source] ?? currentLogLevelSettings['_other'];
+    if (allowedLevel >= LogLevel.Debug) {
+      console.error(`[${this._source} debug]`, ...args)
```

Comment:
Should this be `console.error`, or `debug`?

Reply:
Good catch, fixed.